src/__tests__/createAndQueryUser.ts

```diff
@@ -35,14 +35,12 @@ describe('create and query new user', () => {
     }
   });
 
-  it('`viewer` inaccessible for unauthenticated users', async () => {
+  it('`viewer` is `null` for unauthenticated users', async () => {
```

Comment:
Nit: I wonder if we should use `test` instead of `it` for tests that don't read correctly with `it`

Reply:
Hmm, `eslint-plugin-jest` is currently configured to require a consistent test function name (either `it` or `test`)... so not sure :thinking: